### PR TITLE
Ks/test doc bullets list  DO NOT MERGE THIS PR

### PR DESCRIPTION
**Pull request by KSchopmeyer**
Test Only. DO not merge to master

### CODE REVIEW

File: rtd-requirements.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@ PyYAML>=3.13
 Sphinx>=1.7.6,<2.0.0; python_version == '2.7'
 Sphinx>=3.5.4,!=4.0.0; python_version >= '3.5'
 docutils>=0.13.1,<0.17; python_version == '2.7'
-docutils>=0.13.1; python_version >= '3.5'
+# Test with max limitation for issue #1218, bullets-list
+docutils>=0.13.1,<0.17;; python_version >= '3.5'
```
Comment (andy-maier) [inline rtd-requirements.txt:12]: One semicolon too much

**Review by andy-maier — CHANGES_REQUESTED** [review]
This PR contains two changes: The fix for the bullets in one commit, and general docs improvements in another commit. I think these commits should ideally be in separate PRs, and the commit with the bullet fix should undo the change to the test envs.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -37,52 +37,15 @@ jobs:
       run: |
```
[the remark this answers is not in the document]
Comment (KSchopmeyer) [reply .github/workflows/test.yml:37]: No0te that this is just a test of our issue with docs.  I wanted to make the work of the pr as limited as possible.  The real goal is just to kick off the rtd build.

**Comment by andy-maier** [comment]
Superseded by PR #1221 , closing it.

**Pull request closed without merging**